### DISCUSSION TITLE
Full Icon Set

### DISCUSSION THREAD

**Discussion by augiethehuman** [Shopify/polaris-react#10345] [Ideas]
It would be amazing to have an icon set similar to Polaris, or if Polaris itself expanded to cater to external use. Currently, most consistent icon packs follow a 24px grid with 2px padding. However, there is a growing trend among companies like Linear, Attio, and Shopify to adopt a system with 0px padding on a smaller pixel grid. If anyone knows of such an icon set, excluding Streamline which lacks attention to detail, kindly inform me.

**Comment by yurm04** [comment]
Hey @augiethehuman we have a `polaris-icons` package that includes all of the icons in our design system and can be consumed in your projects

**Reply by augiethehuman** [reply]
Hey there! What I meant was a full solid set of icons.

**Reply by yurm04** [reply]
Thanks for that clarification. We've just recently redrawn our icon set to fit the new design language shipped with Polaris v12. As part of our design language [we use filled and outlined icons](https://polaris.shopify.com/design/icons/creating-icons#filled-vs-outlined) for different purposes, so it's not likely we will be able to provide an additional set of only filled icons
